pkg/handler/processor/guesser/format_ite6.go

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package guesser
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/guacsec/guac/pkg/handler/processor"
+)
+
+type attestation struct {
```

Comment:
hmm, maybe use the intoto library here for the statement, like https://github.com/guacsec/guac/pull/39/files#diff-57f9f62d5b3797cdf1646065bad24ebaeed1d19eeb717a918f7b7e0f60623e5fR82